backend/src/ml_space_lambda/utils/common_functions.py

```diff
@@ -149,7 +149,6 @@ def get_tags_for_resource(sagemaker, arn: str):
 
 def list_clusters_for_project(
```

Comment:
Is there a reason this wasn't switched over to use resource metadata and is still relying on the EMR client?

Reply:
The only place we're using it at this point is when we create the EMR cluster. We grab the list of clusters created in the last 20 seconds then we create an entry in the resource scheduler for each one of them.

```
        clusters = list_clusters_for_project(
            emr=emr,
            created_after=datetime.datetime.fromtimestamp(time.time() - 20),
        )
        for cluster in clusters["records"]:
            if cluster["Name"] == cluster_name:
                resource_scheduler_dao.create(
                    ResourceSchedulerModel(
                        resource_id=cluster["Id"],
                        resource_type=ResourceType.EMR_CLUSTER,
                        termination_time=termination_time,
                        project=project_name,
                    )
                )
                break
```

I'm not positive that this EMR cluster would exist in the resource metadata table yet since we'd have a run between the `run_job_flow` execution triggering the CW event logic, versus hitting this line.

Reply:
Ok I forgot about that. `run_job_flow` returns a cluster arn but not a cluster id and we can't describe a cluster without the id so that's what we use for termination. Seems like we need to just leave this for now.